typescript/src/updater/update_feed.ts

```diff
@@ -0,0 +1,171 @@
+// ## Cli
+// - chain
+// - target-chain (de Pragma.sol - donc ça doit être déployé)
+// - feed_id
+// - private_key
+// - (keystore)
+
+// ## Steps
+// 1. Theoros ? (choix de l'endpoint)
+// 2. Get calldata from Theoros
+// 3. Call `updateDataFeeds` avec la calldata
+// 4. Assertions - check that on the destination chain everything is correctly updated
+//    Show gas consumption
+
+import { Command } from "commander";
+import { ethers } from "ethers";
+import abi from "../../solidity/out/Pragma.sol/Pragma.json";
+
+interface ChainConfig {
+  contractId: string;
+}
+
+function getChainConfig(chainId: string): ChainConfig {
+  const chainConfigs: { [key: string]: ChainConfig } = {
```

Comment:
This should be a configuration in a separate yaml file I think. And we just read the config there.

Maybe we can read directly from the `deployments` folder? And so if something isn't deploy we can't update it?

Reply:
done i guess

Reply:
Almost, the output deployment format is `deployments/[chain]/[oracle | dispatcher | pragma].json`

`pragma.json` will have the sol deployed contract like:
```json
{
"Pragma": "0x..."
}

Reply:
done